app/templates/components/event-info.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="ui segments">
+  <div class="ui secondary segment">
+    <h3 class="weight-400">{{t 'When & Where'}}</h3>
+  </div>
+  <div class="ui padded segment">
+    <p>
+      {{t 'freiLand Friedrich-Engels-StraBe 22 Postdam Germany 14473'}}
+      <br>
+      {{t 'Thursday, May 25,2017 at 09:00 AM - Thursday, May 25,2017 at 09:00 PM (Europe/Berlin)'}}
+    </p>
+  </div>
+</div>
```

Comment:
@mariobehling same reason. Since not dealing with the API in this particular PR.